examples/plot_geodesics_h2.py

```diff
@@ -0,0 +1,35 @@
+"""
+Plot a geodesic on the hyperbolic space h2,
```

Comment:
could you potentially add one or a few renderings to this or a subsequent CL? I'm curious :)

Reply:
<img width="533" alt="screen shot 2018-02-21 at 11 39 52 am" src="https://user-images.githubusercontent.com/8267869/36475754-fcc8d93c-16fb-11e8-9f78-2354bedd780c.png">

The curvature increases next to the disk center and the points get closer as the geodesic approches the disk boundary, which makes sense as the disk's boundary represents the infinity, the boundary is never reached.